src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -279,6 +289,13 @@ public interface IDurableOrchestrationClient
         /// <returns>Returns an instance of <see cref="PurgeHistoryResult"/>.</returns>
         Task<PurgeHistoryResult> PurgeInstanceHistoryAsync(string instanceId);
 
+        /// <summary>
+        /// Purge the history for multiple instances.
+        /// </summary>
+        /// <param name="instanceIds">The IDs of the orchestration instance to purge.</param>
+        /// <returns>>Returns an instance of <see cref="PurgeHistoryResult"/>.</returns>
+        Task<PurgeHistoryResult> PurgeInstanceHistoryAsync(IEnumerable<string> instanceIds);
+
```

Comment:
I suggest removing the extra `>` in the `returns` tag. Also a small wording suggestion.
```suggestion
        /// <summary>
        /// Purge the histories of multiple instances.
        /// </summary>
        /// <param name="instanceIds">The instanceIDs whose histories to purge.</param>
        /// <returns>A <see cref="PurgeHistoryResult"/> for the aggregate purge operation.</returns>
        Task<PurgeHistoryResult> PurgeInstanceHistoryAsync(IEnumerable<string> instanceIds);

```